client/src/components/OptionsMenu.js

```diff
@@ -27,6 +27,12 @@ const OptionsMenu = ({ onDelete, onEdit }) => {
                                         <Text>Edit</Text>
                                     </MenuItem>
                                 )}
+                                {onClear && (
+                                    <MenuItem onClick={onClear}>
+                                        <Icon name="close" mr={3} />
+                                        <Text>Clear plan</Text>
+                                    </MenuItem>
+                                )}
```

Comment:
Instead of passing a separate function, can we just use `onDelete` instead? It essentially does the same thing right?

Reply:
OnDelete deletes the plan. Is onclear supposed to occur for a student plan or programme plan

Reply:
If you delete a default plan (programme), doesn't that essentially clear it?

Reply:
onClear should be for a programme plan (default template)

Reply:
Ooh.  I thought the function was supposed to remove all course allocations in a student plan. So it just deletes a programme plan?

Reply:
Shouldn't he also be able to clear any plan, not just the template?

Reply:
yea its for all plans